CONTRIBUTION.md

```diff
@@ -0,0 +1,184 @@
+# Contributing to Antenna
+
+First off, thanks for contributing!
+
+This is a set of guidelines for contributing to Antenna, which is hosted by the [Eclipse Foundation](https://github.com/eclipse) on Github.
```

Comment:
Maybe link "Antenna" directly to the GitHub project, similar to the link to the Eclipse org on GitHub?